.github/workflows/example.yml

```diff
@@ -27,19 +27,22 @@ defaults:
     shell: bash
 
 jobs:
-  
-  # These jobs run when creating a pull request
-  plan-development:
-    name: Plan Development - example
+
+  plan-dev-test:
+    strategy:
+      matrix:
+        include:
+          - environment: development
+    name: Terraform plan - ${{ github.workflow }} - ${{  matrix.environment }}
     runs-on: ubuntu-latest
     if: github.ref != 'refs/heads/main' || github.event_name == 'workflow_dispatch'
     env:
-      TF_ENV: development
+      TF_ENV: ${{ matrix.environment }}
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v3.0.2
       - name: Set Account Number
-        run: echo "ACCOUNT_NUMBER=$(jq -r -e --arg account_name "example-${TF_ENV}" '.account_ids[$account_name]' <<< $ENVIRONMENT_MANAGEMENT)" >> $GITHUB_ENV
+        run: echo "ACCOUNT_NUMBER=$(jq -r -e --arg account_name "${GITHUB_WORKFLOW}-${TF_ENV}" '.account_ids[$account_name]' <<< $ENVIRONMENT_MANAGEMENT)" >> $GITHUB_ENV
```

Comment:
I like the use of this variable 👏

Reply:
The holy grail would be to use it in the paths and have an almost completely generic workflow file but I can't figure out how. :(